detail.scss

```diff
@@ -0,0 +1,51 @@
+.title {
+  width: 100%;
+  color: #000;
+  font-family: sans-serif;
```

Comment:
I think the font-family is Roboto 🤔 . Check Google Fonts out! 😄

Reply:
Done